src/next-collection-view.js

```diff
@@ -71,10 +72,16 @@ const CollectionView = Backbone.View.extend({
   },
 
   // Create an region to show the emptyView
-  _initEmptyRegion() {
-    this.emptyRegion = new Region({ el: this.el, replaceElement: false });
+  getEmptyRegion() {
+    if (this._emptyRegion && !this._emptyRegion.isDestroyed()) {
+      return this._emptyRegion;
+    }
+
+    this._emptyRegion = new Region({ el: this.el, replaceElement: false });
+
```

Comment:
replaceElement is false by default, so does not need it here

Reply:
the reason it is here is because replaceElement: true would totally break the collectionview. It can never be true here. So if someone changes the default on the region prototype collectionview would break.